docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -708,6 +708,16 @@ WITH (
 )
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
+(delta-lake-shallow-clone)=
+#### Shallow clone
+
+The connector supports reading from shallow cloned tables. A shallow clone is a
```

Comment:
Maybe we need to emphasize that only reading is supported.

Also ... importantly .. we do NOT support anything like creating those tables.. we have to point that out.. otherwise people will go to the linked page .. copy a statement like 

CREATE TABLE target_table SHALLOW CLONE source_table;

and wonder why it isnt working.